Formula/create-api.rb

```diff
@@ -0,0 +1,29 @@
+class CreateApi < Formula
+  desc "Delightful code generator for OpenAPI specs"
+  homepage "https://github.com/CreateAPI/CreateAPI"
+  url "https://github.com/CreateAPI/CreateAPI/archive/refs/tags/0.0.5.tar.gz"
+  sha256 "c250ff140af83e093d86fef0dd18b87363ae91a087c0804bedb40f9922989093"
+  license "MIT"
+  head "https://github.com/CreateAPI/CreateAPI.git", branch: "main"
+
+  depends_on xcode: ["13.0", :build]
+
+  uses_from_macos "swift"
+
+  resource("test-data") do
```

Comment:
```suggestion
  resource "homebrew-test-data" do
```

This way CI knows the resources are internal to homebrew.

Reply:
Thanks @SMillerDev, updated 👍